lib/show.rb

```diff
@@ -0,0 +1,14 @@
+class Show
+  attr_reader :name, :creator, :characters
+
+    def initialize(name, creator, characters)
+      @name = name
+      @creator = creator
+      @characters = characters
+    end
+
+    def total_salary
+      characters.inject(0) {|salary, character| salary + character.salary}
```

Comment:
this is good, you can also use `.sum` here:
`@characters.sum {|char| char.salary}`